### ISSUE TITLE
Possible encoding error with French translation

### ISSUE THREAD

**Issue by ocdtrekkie** [sandstorm-io/sandstorm#3065]
Reported via IRC, I'm not sure how to troubleshoot it.

![image](https://user-images.githubusercontent.com/4399499/39385348-c9d17dd2-4a35-11e8-8330-d6a981f189ec.png)

**Comment by zenhack** [comment]
Hm I tried setting my language settings to fr_FR, with both UTF-8 and ISO-8859-1, and was unable to reproduce this. Question: is the server reporting the encoding of the document in the response headers? I've seen stuff that looks like this with utf8 encoded text when the encoding is missing from both the response headers and the `<head>` element. It looks like sandstorm doesn't put the needed meta tag in the document itself; maybe the server is somehow misconfigured and not sending back the correct charset value in the Content-Type header?

**Comment by kentonv** [comment]
I agree it looks like what happens if you send UTF-8 but the browser expects ISO-8859-1.

I think this may be a problem with the way translation tables are transferred. E.g.

https://oasis.sandstorm.io/tap-i18n/fr.json

If you look at the headers, it actually has:

    content-type: text/html

Now, obviously, this is totally wrong. But also obviously, it's not being parsed as HTML. So the question is, what parses it, and why does it parse as UTF-8 for some people and not everyone.

It looks like tap-i18n uses jQuery's `getJSON()`. I guess on Chrome, this function must somehow end up parsing the response as UTF-8, but on some other browser, it gets parsed as ISO-8859-1. Unfortunately the documentation for `getJSON()` does not bother to mention how it decides what charset to use. Maybe from the content-type? But why only on some browsers?

**Comment by zenhack** [comment]
Huh. if I load an actual sandstorm ui page it sends back `content-type: text/html; charset=utf-8`.

My browser (both firefox and chromium) does try to render the JSON as html. It similarly seems to try to interpret it as ISO-8859-1.

We could just add a meta tag to `shell.html`, which would likely sidestep the whole issue of the http headers.

**Comment by zenhack** [comment]
Ah, wait, hadn't realized the translations are actually processed client-side; meta tag probably won't help then.

**Comment by zenhack** [comment]
The Content-Type header for the JSON seems to be being set here:

https://github.com/iron-meteor/iron-router/blob/devel/lib/router_server.js#L64

I tried adding the charset parameter to that in the generated bundle for my local sandstorm dev server, and it started sending it in the response bodies.

I assume that's some kind of catchall?
